src/VisualStudio/Core/Next/Remote/ServiceHubRemoteHostClient.cs

```diff
@@ -73,28 +73,16 @@ internal sealed partial class ServiceHubRemoteHostClient : RemoteHostClient
                 return;
             }
 
-            // don't block UI thread while initialize workspace host
-            var host = new WorkspaceHost(vsWorkspace, client);
-
-            // Initialize the remote side with whatever data we have currently for the workspace.
-            // As workspace changes happen, this host will get notified, and it can remote those
-            // changes appropriately over to the remote size.
-            await host.InitializeAsync().ConfigureAwait(false);
```

Comment:
@heejaechang I looked into keeping the session around for the lifetime of the client.  However, it wasn't clear to me what the lifetime of the client is.  There is 'Shutdown' 'Disconnect' and 'Reconnect', and it wasn't clear to me which of those would necessitate a new session being created.